make_plant_parts_eia.py

```diff
@@ -1,4 +1,57 @@
+with records that correspond to mostly generators and plants. Practically
+speaking, a plant is a collection of generator(s). There are many attributes
+of generators (i.e. prime mover, primary fuel source, technology type). We can
+use these generator-based attributes to create other records about other
+collections of plant parts. A "plant part" record which cooresponds to a
+particular collection of generator attributes (i.e. all of the generators with
+unit id "2").
+
+Because generators are often owned by multiple utilities, another dimention of
+the master unit list involves generating two records for each owner: one of the
+portion of the plant they own and one for the plant as a whole. The portion
+records are labeled in the `ownership` column as "owned" and the total records
+are labeled as "total".
+
+This module refers to "true granularies". Many plant parts we cobble together
+here in the master unit list refer to the same collection of infrastructure as
+other master unit list records. For example, if we have a "plant_prime_mover"
+plant part record and a "plant_unit" plant part record which were both cobbled
+together from the same two generators. We want to be able to reduce the master
+unit list to only unique collections of generators, so we label the first
+unique granularity as a true granularity and label the subsequent records as
+false granularities with the `true_gran` column. In order to choose which plant
+part to keep in these instances, we assigned a `plant_parts_ordered` and
+effectively keep the first instance of a unique granularity.
+
+Overview of flow for generating the master unit list:
+
+There are two classes in here - one which compiles input tables (CompileTables)
+and one which compiles the master unit list (CompilePlantParts). The method
+that rules the show here is `generate_master_unit_list`, which is a method of
+CompilePlantParts.
+
+`PLANT_PARTS` is basically the main recipe book for how each of the plant parts
+need to be compiled. CompilePlantParts eats this recipe book and follows the
+recipe to make the master unit list.
+
+All of the plant parts are compiled from generators. So we first generate a
+big dataframe of generators with any columns we'll need. This is where we add
+records regarding utility ownership slices. Then we use that generator
+dataframe and information stored in `PLANT_PARTS` to know how to aggregate each
+of the plant parts. Then we have plant part dataframes with the columns which
+identify the plant part and all of the data columns aggregated to the level of
+the plant part.
+
+With that compiled plant part dataframe we also add in qualifier columns with
+`get_qualifiers`. A qualifer column is a column which contain data that is not
+endemic to the plant part record (it is not one of the identifying columns or
+aggregated data columns) but the data is still useful data that is attributable
+to each of the plant part records.
```

Comment:
This is kind of incomplete / confusing, but you go into it in more detail below. Maybe just give a pointer to the other method for more information. "For more detail on what a qualifier column is, see the `get_qualifiers()` method.

Reply:
the `get_qualifiers` method is referenced in the first sentence here.... I can certainly include this extra pointer, but it feels duplicative